plugins/GSdx/GSDialog.cpp

```diff
@@ -164,12 +164,27 @@ void GSDialog::SetTextAsInt(UINT id, int i)
 	SetText(id, buff);
 }
 
-void GSDialog::ComboBoxInit(UINT id, const vector<GSSetting>& settings, int32_t selectionValue, int32_t maxValue)
+void GSDialog::ComboBoxInit(UINT id, const vector<GSSetting> settings, int32_t selectionValue, int32_t maxValue)
 {
+	if (settings.empty()) return;
+
 	HWND hWnd = GetDlgItem(m_hWnd, id);
+	bool currentsel_found = false;
 
 	SendMessage(hWnd, CB_RESETCONTENT, 0, 0);
 
+	for (const GSSetting i : settings)
```

Comment:
> std::find_if could be used instead of a for loop.

Unless I'm wrong, I'd also have to define an unary function for the third parameter of ``find_if()`` in that case, is that fine?